shared/src/lib.rs

```diff
@@ -458,8 +458,8 @@ impl Default for Scaling {
         Scaling {
             scale_type: ScaleType::FixedNumSegments,
             min_num_segments: 1,
-            scale_factor: 1,
-            target_rate: 1000,
+            scale_factor: 0,
```

Comment:
Is 0 a valid number here?

Reply:
It is a no-op since the scaling type is `ScaleType::FixedNumSegments`.
I have changed it to 1.